api/types/webauthn/webauthn.proto

```diff
@@ -49,6 +49,19 @@ message SessionData {
     bytes user_id = 2 [ (gogoproto.jsontag) = "userId,omitempty" ];
     // Raw Credential IDs of the credentials allowed for the ceremony.
     repeated bytes allow_credentials = 3 [ (gogoproto.jsontag) = "allowCredentials,omitempty" ];
+    // True if resident keys were required by the server / Relying Party.
+    bool resident_key = 4;
+    // Requested user verification requirement, either "discouraged" or
+    // "required".
+    // An empty value is treated equivalently to "discouraged".
+    string user_verification = 5;
```

Comment:
Thus values are know maybe enum type will fit better here ?

Reply:
I like the more "opaque" value here, since we don't own the underlying protocol (WebAuthn).

Reply:
All right, I'm ok with current version. We call always use `casttype` in the future if needed.